### ISSUE TITLE
Allow "Partial" `Import Code Snippet`

### ISSUE THREAD

**Issue by jjangga0214** [vuejs/vuepress#2118]
#### What problem does this feature solve?

`Import Code Snippet` feature is cool. Users don't have to copy and paste code from the original file. It does not require "syncing" when the the referred file is modified.  

However, in most cases, I'd want to import just a few lines from the source code. Because the entire file would be obviously "long" in many cases, and contains "unnecessary" content. 

#### What does the proposed API look like?

Let's assume I want to only import from line 5 to 10.

##### Proposal 1

```
<<< [5-10] @/filepath
```

##### Proposal 2

```
<<< @/filepath [5-10] 
```

##### Proposal 3

```
<<< @/filepath[5-10] 
```

#### Are you willing to work on this yourself?

Not sure right now.

**Comment by meteorlxy** [comment]
See #1336

For now, you can try this markdown-it plugin: https://github.com/fabbballin/markdown-it-vuepress-code-snippet-enhanced

**Comment by billyyyyy3320** [comment]
Resolved by #2225
